Charts/Classes/Charts/PieChartView.swift

```diff
@@ -25,6 +25,7 @@ public class PieChartView: PieRadarChartViewBase
     private var _circleBox = CGRect()
     
     private var _drawXLabelsEnabled = true
+    private var _drawXLabelsMinimumAngle: Double = 0
```

Comment:
Good point! I could have sworn that I used a `Double` here for a reason, but I can't remember that reason now, so I've switched it to a `CGFloat`.